src/others/helpers/map-utils.test.ts

```diff
@@ -3,250 +3,259 @@ import { assignTotalForDate, groupLocationGroupByDate, mapAidRequestsToFeatures,
-            },
-            {
-                type: "Feature",
-                properties: {
-                    amount: 4,
-                    category: "ALL",
-                    date: "2022-03-10",
-		    normalized_amount: 0.4,
-		    description: "\nPersonal hygiene kits: 4",
-                },
-                geometry: { type: "Point", coordinates: [36.169, 47.645] },
-            },
-            // Location Kyiv
-            // Features for single requests
-            {
-                type: "Feature",
-                properties: {
-                    amount: 1,
-                    category: "Batteries",
-                    date: "2022-03-11",
-		    normalized_amount: 0.5,
-		    description: "Batteries: 1",
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            },
-            {
-                type: "Feature",
-                properties: {
-                    amount: 2,
-                    category: "Baby Products",
-                    date: "2022-03-10",
-		    normalized_amount: 1,
-		    description: "Baby Products: 2",
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            },
-            {
-                type: "Feature",
-                properties: {
-                    amount: 3,
-                    category: "Sanitary pads",
-                    date: "2022-03-10",
-		    normalized_amount: 0.375,
-		    description: "Sanitary pads: 3",
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            },
-            {
-                type: "Feature",
-                properties: {
-                    amount: 5,
-                    category: "Personal hygiene kits",
-                    date: "2022-03-10",
-		    normalized_amount: 1,
-		    description: "Personal hygiene kits: 5",		    
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            },
-            // Features for aggregated requests for day and location
-            {
-                type: "Feature",
-                properties: {
-                    amount: 1,
-                    category: "ALL",
-                    date: "2022-03-11",
-		    normalized_amount: 0.1,
-		    description: "\nBatteries: 1",
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            },
-            {
-                type: "Feature",
-                properties: {
-                    amount: 10,
-                    category: "ALL",
-                    date: "2022-03-10",
-		    normalized_amount: 1,
-		    description: "\nPersonal hygiene kits: 5\nSanitary pads: 3\nBaby Products: 2",
-                },
-                geometry: { type: "Point", coordinates: [30.524, 50.45] },
-            }
-        ]));
+  expect(new Set(mapAidRequestsToFeatures(decodedAidRequestGroupedByLocation))).toEqual(
```

Comment:
mainly `prettier` fixes to this file, but also updated the `expect` to include `city` here